app/package.json

```diff
@@ -46,6 +46,7 @@
     "@tiptap/react": "^2.1.10",
     "@tiptap/suggestion": "^2.1.10",
     "clsx": "^2.1.0",
+    "date-fns": "^2.x.x",
```

Comment:
It's not good:
1. The most recent version 2.x.x was updated 2 years ago.
2. It is better to use the exact version of the package, because they can roll out radically different changes and break our production build.